CHANGELOG.MD

```diff
@@ -1,22 +1,4 @@
 # Changelog
 
-This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
-
-## [Unreleased]
-
-### New features
-
-## [0.1.1] - 2020-08-14
-
-### Bugfixes
-
-- The code wasn't shipped properly when publishing a non-dev release to NPMJS.
-
-## [0.1.0] - 2020-08-10
-
-### New features
-
-First release! What's possible with this first release:
-
-- Authenticate a Web app to a Solid Identity provider
-- Perform an authenticated fetch to a Pod Server, using a DPoP token
+This change log is intended to record changes to the mono-repo as a whole, and
```

Comment:
Hhhmmm... I could go either way on this one (or I could just a changelog generator: https://github.com/lerna/lerna-changelog?). I suppose it would be good to have consistency across the whole company though for mono-repos, so I'll bring it up with Jack next week.

Reply:
I hate changelog generators! :P That's how you get [these useless changelogs](https://github.com/inrupt/solid-client-js/pull/415):

<blockquote>
<h2>v7.8.1</h2>
<ul>
<li><a href="https://github.com/eslint/eslint/commit/f542b5d0679b73326ad249fc44a54c3f848bd3e6"><code>f542b5d</code></a> Fix: Update broken @eslint/eslintrc version (fixes <a href="https://github-redirect.dependabot.com/eslint/eslint/issues/13641">#13641</a>) (<a href="https://github-redirect.dependabot.com/eslint/eslint/issues/13647">#13647</a>) (Nicholas C. Zakas)</li>
<li><a href="https://github.com/eslint/eslint/commit/c1b56966c2354e12d16e8394443de49fa54f4290"><code>c1b5696</code></a> Sponsors: Sync README with website (ESLint Jenkins)</li>
<li><a href="https://github.com/eslint/eslint/commit/8ddeda01afdb1e9656a43853b8e25c9c4582e6ad"><code>8ddeda0</code></a> Sponsors: Sync README with website (ESLint Jenkins)</li>
<li><a href="https://github.com/eslint/eslint/commit/e02e2fe019a1ed9a34a7b96e4c8961c35093b0ce"><code>e02e2fe</code></a> Sponsors: Sync README with website (ESLint Jenkins)</li>
</ul>
</blockquote>